package.json

```diff
@@ -43,6 +43,7 @@
   "dependencies": {
     "@hapi/cryptiles": "5.0.0",
     "@hapi/wreck": "^17.1.0",
+    "debug": "^4.3.4",
```

Comment:
yarn upgrade auto-added this entry

Reply:
I don't think this is right. We are not using debug, it is a transitive dependency, so it shouldn't be added here. We already have a resolution for it.